projects/1.16/assets/ars-nouveau/ars_nouveau/lang/zh_cn.json

```
@@ -1,848 +1,883 @@
+	"ars_nouveau.spell_book_gui.familiar": "使魔",
+	"ars_nouveau.spell_book_gui.close": "关闭",
+	"effect.ars_nouveau.familiar_sickness": "使魔召唤失调",
+	"ars_nouveau.familiar.sickness": "召唤另一个使魔前，你必须等一段时间。",
+	"ars_nouveau.gui.notebook": "说明文档",
+	"ars_nouveau.gui.color": "颜色选择器",
+	"ars_nouveau.gui.familiar": "使魔",
+	"entity.ars_nouveau.bookwyrm": "书龙",
+	"ars_nouveau.category.familiars": "使魔能提供被动增益，也能协助战斗。要了解更多信息，请参阅使魔召唤相关条目。",
+	"ars_nouveau.page1.summoning_familiars": "召唤使魔",
+	"ars_nouveau.page2.summoning_familiars": "正式开始召唤使魔之前，你需要先获得你想召唤的实体对应的绑定手稿。这种手稿可通过在相关实体附近进行绑定仪式得到。可作为使魔的相关实体列表能在使魔一章中找到。获得手稿后只需使用手稿就能学到使魔的召唤方法。",
+	"ars_nouveau.page3.summoning_familiars": "得到绑定手稿后，你就能从你法术书的法术合成界面中找到使魔列表。选中一个使魔就能将其召唤到世界中，同时给予召唤者使魔召唤失调，使其短期内不能召唤另一个使魔。对于第一个使魔，建议在卡班克尔旁进行绑定仪式。使魔与玩家绑定，无法在书间转移。",
+	"ars_nouveau.page4.summoning_familiars": "通常情况下只能同时召唤一个使魔，召唤另一个使魔会驱散先前的那个。你可以通过对自己的使魔施放解除魔法来驱散它。作为给予你增益的代价，使魔存在时会从主人处夺取一部分魔力上限。",
+	"ars_nouveau.familiar.owned": "你已经拥有了这个使魔。",
+	"ars_nouveau.familiar.unlocked": "已解锁使魔。",
+	"ars_nouveau.familiar.script": "用于获得使魔。来自于绑定仪式。",
+	"ars_nouveau.schools": "学派：",
+	"entity.ars_nouveau.familiar_bookwyrm": "书龙",
+	"entity.ars_nouveau.familiar_carbuncle": "卡班克尔",
+	"entity.ars_nouveau.familiar_sylph": "希尔芙",
+	"entity.ars_nouveau.familiar_wixie": "薇克精",
+	"entity.ars_nouveau.familiar_drygmy": "德格米",
+	"ars_nouveau.mirror.invalid": "无效法术。巫师之镜的法术只能含效果类和增强类构件。",
+	"item.ars_nouveau.enchanters_mirror": "巫师之镜",
+	"ars_nouveau.page.enchanters_mirror": "对使用者自身施展法术。若通过这面镜子施法，任何魔符的魔力消耗都会减少，持续时间也会更长。需要通过抄写台写入不含形态的法术，如力量 -> 治疗。",
+	"block.ars_nouveau.bookwyrm_lectern": "书龙讲台",
+	"ars_nouveau.seconds": "%s 秒",
+	"block.ars_nouveau.basic_spell_turret": "基础法术炮塔",
+	"block.ars_nouveau.spell_turret": "附魔法术炮塔",
+	"block.ars_nouveau.timer_spell_turret": "定时法术炮塔",
+	"ars_nouveau.page.spell_turret":"炮塔能在通入红石信号时自动施法，就像发射器一样。炮塔施放的法术可以通过抄好的法术羊皮纸设置，而且只接受触摸和弹射物形态的法术。施法时，炮塔会从附近魔源罐抽取魔源。只要炮塔旁有物品容器，炮塔就能有拾取物品和放置方块的能力。",
+	"ars_nouveau.page.spell_turret2": "附魔法术炮塔施法时的魔源消耗是基础炮塔的一半。",
+	"ars_nouveau.page.spell_turret3": "定时法术炮塔会定时自动施法。默认施法间隔为 1 秒，而且能通过右击炮塔增加。左键点击炮塔则会减少这一间隔。潜行时，每次点击增加/减少的时间间隔增大到 10 秒。对炮塔使用支配之杖可以锁定间隔，防止意外修改。将间隔改至 0 秒或者通入红石信号会关闭炮塔。炮塔施放法术中的弹射物、触摸和红石信号构件不需要魔源。",
+	"ars_nouveau.page.spell_turret4":"炮塔能用于实现紧凑、高效的自动化，例如：可配置红石时钟、单方块林场/农田、带时运的高速熔炼、带掠夺的自动刷怪场。",
+	"effect.ars_nouveau.bounce": "弹性",
+	"ars_nouveau.carbuncle.path": "如果路上有这种方块，卡班克尔会倾向于走在上面。",
+	"ars_nouveau.carbuncle.pathing": "倾向于走在%s上",
+	"ars_nouveau.summoning": "召唤",
+	"ars_nouveau.item_transport": "物品运输",
+	"ars_nouveau.filtering": "过滤",
+	"ars_nouveau.pathing": "寻路",
+	"block.ars_nouveau.archwood_chest": "至高木箱子",
+	"ars_nouveau.page1.archwood_forest": "至高木森林是一种略稀有的生物群系，充满着魔法光源和至高木。至高木森林中金矿和魔源宝石矿的生成比其他群系更多。另外，卡班克尔、希尔芙和德格米等魔法生物在其中生成的几率也比其他地方更高。",
+	"ars_nouveau.wilden": "荒野族裔",
+	"ars_nouveau.decorative": "装饰性方块",
+	"ars_nouveau.archwood_trees": "至高木",
+	"block.ars_nouveau.spell_prism": "法术棱镜",
+	"ars_nouveau.page.spell_prism": "射中该方块的弹射物会被转向，从方块正面离开。法术转向时，法术棱镜会向附近的侦测器发出信号。"
```

|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
	"ars_nouveau.page.spell_prism": "当弹射物法术射向此方块时，该法术会被重定向，向着此方块的面朝方向射出。法术转向时，法术棱镜会向附近的侦测器发出信号。"
```